packages/zosfiles/__tests__/__unit__/methods/create/Create.unit.test.ts

```diff
@@ -251,6 +251,38 @@ describe("Create data set", () => {
             );
         });
 
+        it("should be able to create a sequential data set using the a block size that is too small", async () => {
```

Comment:
Would it be good to add more of these tests to ensure that the other RECFM options specified in the case statement also behave the same? It says the only statement that isn't covered is the RECFM being "V", which is a bit odd.